libdnf/dnf-context.c

```diff
@@ -1420,24 +1419,17 @@ dnf_context_setup_enrollments(DnfContext *context, GError **error)
     if (getuid () != 0)
         return TRUE;
 
-    for (i = 0; cmds[i] != NULL; i++) {
-        int child_argc;
-        g_auto(GStrv) child_argv = NULL;
-        int estatus;
+#ifdef RHSM_SUPPORT
+    g_autoptr(RHSMContext) rhsm_ctx = rhsm_context_new ();
```

Comment:
Aren't we going to run into problems backporting this since EL7's glib is too old to auto-define autoptrs for types?